@xen-orchestra/lite/src/views/pool/PoolTasksView.vue

```diff
@@ -1 +1,69 @@
-<template>Tasks (coming soon)</template>
+<template>
+  <UiCard>
+    <UiTitle class="title-with-counter" type="h4">
+      {{ $t("tasks") }}
+      <UiCounter :value="pendingTasks.length" color="info" />
+    </UiTitle>
+
+    <TasksTable :pending-tasks="pendingTasks" :finished-tasks="finishedTasks" />
+    <UiSpinner class="loader" v-if="!isReady" />
+  </UiCard>
+</template>
+
+<script lang="ts" setup>
+import TasksTable from "@/components/tasks/TasksTable.vue";
+import UiCard from "@/components/ui/UiCard.vue";
+import UiCounter from "@/components/ui/UiCounter.vue";
+import UiSpinner from "@/components/ui/UiSpinner.vue";
+import UiTitle from "@/components/ui/UiTitle.vue";
+import useArrayHistory from "@/composables/array-removed-items-history.composable";
+import useCollectionFilter from "@/composables/collection-filter.composable";
+import useCollectionSorter from "@/composables/collection-sorter.composable";
+import useFilteredCollection from "@/composables/filtered-collection.composable";
+import useSortedCollection from "@/composables/sorted-collection.composable";
+import type { XenApiTask } from "@/libs/xen-api";
+import { useTaskStore } from "@/stores/task.store";
+import { useTitle } from "@vueuse/core";
+import { storeToRefs } from "pinia";
+import { computed } from "vue";
+
+const { allRecords, isReady } = storeToRefs(useTaskStore());
+
+const { compareFn } = useCollectionSorter<XenApiTask>({
+  initialSorts: ["-created"],
+});
+
+const allTasks = useSortedCollection(allRecords, compareFn);
+
+const { predicate } = useCollectionFilter({
+  initialFilters: ["!name_label:|(SR.scan host.call_plugin)", "status:pending"],
+});
+
+const pendingTasks = useFilteredCollection<XenApiTask>(allTasks, predicate);
+
+const tasksHistory = useArrayHistory(allTasks, 500, (task) => task.uuid);
+
+const finishedTasks = computed(() => tasksHistory.value.slice(-10));
```

Comment:
I don't get this, can you explain?

Reply:
I made that to (manually) test pagination and forgot to remove.

`tasksHistory` would contains all finished tasks and `finishedTasks` would contains only the one to be displayed (the naming is really bad too, but was not intended to be published ^^).

For now, we'll keep only the history since pagination is not part of this card.